common/supplemental/windowsZones.xml

```diff
@@ -736,9 +751,13 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<mapZone other="Central Pacific Standard Time" territory="VU" type="Pacific/Efate"/>
 			<mapZone other="Central Pacific Standard Time" territory="ZZ" type="Etc/GMT-11"/>
 
-			<!-- (UTC+12:00) Anadyr, Petropavlovsk-Kamchatsky -->
-			<mapZone other="Russia Time Zone 11" territory="001" type="Asia/Kamchatka"/>
-			<mapZone other="Russia Time Zone 11" territory="RU" type="Asia/Kamchatka Asia/Anadyr"/>
+      <!-- (UTC+12:00) Anadyr, Petropavlovsk-Kamchatsky -->
+      <mapZone other="Russia Time Zone 11" territory="001" type="Asia/Anadyr"/>
+      <mapZone other="Russia Time Zone 11" territory="RU" type="Asia/Anadyr"/>
+
+      <!-- (UTC+12:00) Kamchatka -->
```

Comment:
The `"Kamchatka Standard Time"` zone has been deprecated in Windows, and thus should not appear in the CLDR.  It is marked with `IsObsolete` in the Windows registry.

Any system having reference to `"Kamchatka Standard Time"` should be updated to use `"Russia Time Zone 11"` instead.